_pages/en_US/410.txt

```diff
@@ -0,0 +1,12 @@
+---
+title: "Page no longer exists" #
+sitemap: false
+---
+
+The page you are trying to view used to exist, but it has since been removed because it has been superseded by better exploits. Maybe the instructions you are following are outdated? (If you got here from a link on this guide, [let us know](https://github.com/hacks-guide/Guide_3DS/issues))
```

Comment:
Might want to reword this warning a bit? Idk

Reply:
You wrote this and pushed it to master lol

Idk, the wording seems solid